docs/relnotes.md

```diff
@@ -7,18 +7,66 @@ hide:
 
 This is the list of changes to Taipy releases as they were published.
 
-## Community edition: 1.2 (In progress)
+## Community edition: 2.0
+
+Published on TO_BE_DEFINED.
+
+[`taipy` 2.0](https://pypi.org/project/taipy/2.0.0/) contains the latest
+[`taipy-config` 2.0](https://pypi.org/project/taipy-config/2.0.0/),
+[`taipy-gui` 2.0](https://pypi.org/project/taipy-gui/2.0.0/),
+[`taipy-core` 2.0](https://pypi.org/project/taipy-core/2.0.0/) and
+[`taipy-rest` 2.0](https://pypi.org/project/taipy-rest/2.0.0/) packages.
+
+#### Features
+
+**`taipy-core`** 2.0.0
+
+   - New data node: SQLTableDataNode. It represents a table in a SQL database.
+   - New data node: JSONDataNode. It represents the data from a JSON file.
+   - SQLDataNode behavior is changed due to the release of SQLTableDataNode. Now it represents the data
+     using custom read and write queries.
+   - In standalone mode, a job whose status is `SUBMITTED`, `PENDING`, or `BLOCKED`, can be canceled. When
+     canceling the job, its subsequent jobs will be abandoned and their statuses will be set to `ABANDONED`. When
+     the cancel method is called on a job with its status being either `RUNNING`, `COMPLETED`, or `SKIPPED`, its
+     subsequent jobs will be abandoned while its status remains unchanged. A job whose status is `FAILED`,
+     `CANCELED`, or `ABANDONED` cannot be canceled.
+   - Taipy Core can now be run as a service by using `Core().run()` or `tp.run(Core())`. By running Core as a service, Taipy will initialize the scheduler and the job dispatcher based on the config provided. Taipy Core can also be run along with Taipy Gui or Taipy Rest.
```

Comment:
'Taipy will initialize' -> 'Taipy initializes'
'config provided;  -> 'provided configuration'
'Taipy Core can also be run along with Taipy Gui or Taipy Rest' ->
  'The Taipy Core service can be run along with Taipy Gui or Taipy Rest services' ->